sdk/devcenter/Azure.Developer.DevCenter/samples/Sample_CreateDeleteDevBoxAsync.md

```diff
@@ -1,5 +1,68 @@
+# Azure.Developer.DevCenter samples - Dev Boxes
+
 To use these samples, you'll first need to set up resources. See [getting started](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/devcenter/Azure.Developer.DevCenter/README.md#getting-started) for details.
 
-## <scenario> Asynchronously create, connect to, and delete a Dev Box
+## Get all projects in a dev center
```

Comment:
I am assuming these samples are directly coming from the test/samples directory.

Reply:
Yes - we just hadn't wired those samples into these markdown files yet.

This is to resolve a comment on this Arch Board issue: https://github.com/Azure/azure-sdk/issues/5723